// errorcheck

package main

func main() {
	>5 // ERROR `unexpected >, expecting }`
}
